tests/smoke-test-general.bats

```diff
@@ -344,15 +344,14 @@ services:
 	echo "VIRTUAL_HOST=feaTures.Alpha-beta_zulu.docksal" > .docksal/docksal-local.env
 
 	# Check config (check if local environment variables are used in docksal.yml)
-	run fin config
-	echo "$output" | egrep "VIRTUAL_HOST: features.alpha-beta-zulu.docksal"
-	[[ $(echo "$output" | grep -c "feaTures.Alpha-beta_zulu.docksal") -eq 0 ]]
+	run TERM=dumb fin config
```

Comment:
I guess it would be `TERM=dumb run fin config`

Reply:
that ended up fixing it. thanks @achekulaev